### PR TITLE
Feature/homepage ellipsis

### PR DESCRIPTION
**Pull request by agduncan94**
Max width for two sidebars, also ellipsis for my entries and my orgs.
Screen Shot 2019-12-13 at 9 15 46 AM

On smaller screens right two columns become one column.
Screen Shot 2019-12-13 at 9 16 08 AM

### CODE REVIEW

File: src/app/home-page/home-logged-in/home-logged-in.component.scss [hunk 1]
```diff
@@ -13,10 +13,17 @@ mat-sidenav-container {
...
 }
 
-.side-column {
+.left-side-column {
   background-color: #f6f8fa;
 }
 
+@media screen and (min-width: 992px) {
+  .side-column {
+    max-width: 350px !important;
```
Comment (garyluu) [inline src/app/home-page/home-logged-in/home-logged-in.component.scss]: If it's max 350 and min 350, that's pretty much just width: 350px.  Additionally, I thought we banned pixels.

File: src/app/home-page/widget/entries/entries.component.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.entry-item {
+  white-space: nowrap;
```
Comment (garyluu) [inline src/app/home-page/widget/entries/entries.component.scss]: This is used so often that we should put this somewhere general

File: src/app/home-page/home-logged-in/home-logged-in.component.scss (hunk 1 above)
Comment (agduncan94) [reply src/app/home-page/home-logged-in/home-logged-in.component.scss]: What was the replacement to px?

**Comment by denis-yuen** [comment]
(passing build)

File: src/app/home-page/home-logged-in/home-logged-in.component.scss (hunk 1 above)
Comment (agduncan94) [reply src/app/home-page/home-logged-in/home-logged-in.component.scss]: Used REM

**Comment by agduncan94** [comment]
@coverbeck Do you think simply the repository + optional entry name is enough to identify?

**Comment by agduncan94** [comment]
I am in favour of a more human readable name that can be displayed around the site, though I am not sure if they lack of full path will confuse people.

**Comment by denis-yuen** [comment]
> I am in favour of a more human readable name that can be displayed around the site, though I am not sure if they lack of full path will confuse people.

Depends on the context. 
One concern that users had was that one could intentionally make a workflow that is named similarly to another one. That's why we added namespace to workflows in search (i.e. two workflows with the same name next to each other in search)

**Comment by coverbeck** [comment]
> @coverbeck Do you think simply the repository + optional entry name is enough to identify?

Not sure what the answer is at this point; just wanted to point out what I felt what is a small problem.

**Pull request merged**